lib/request.js

```diff
@@ -1,22 +1,28 @@
 'use strict'
 
-function Request (params, req, query, headers, log) {
+function Request (params, req, query, headers, log, ip, ips, hostname) {
```

Comment:
I think this change will require an update in our types file as well

Reply:
Not the constructor, but the new properties in `Request`  needs to be added.

Reply:
I believe the types need to be added in this type declaration block: https://github.com/fastify/fastify/blob/master/fastify.d.ts#L137

Reply:
> I believe the types need to be added in this type declaration block: https://github.com/fastify/fastify/blob/master/fastify.d.ts#L137

Thank you, @Ethan-Arrowood. Added the property, ips in Request declaration block. b542fb8